spec/tapioca/compilers/dsl/protobuf_spec.rb

```diff
@@ -15,7 +15,7 @@ class Tapioca::Compilers::Dsl::ProtobufSpec < DslSpec
         end
       RUBY
 
-      assert_equal([], gathered_constants)
+      assert_equal([], gathered_constants.reject { |constant| constant.start_with?("Google::Protobuf") })
```

Comment:
I think we can simplify this with something like

```suggestion
      assert(gathered_constants.all? { |constant| constant.start_with?("Google::Protobuf") })
```